core/domain/story_services.py

```diff
@@ -65,6 +67,39 @@ def _migrate_story_contents_to_latest_schema(versioned_story_contents):
         story_schema_version += 1
 
 
+def get_pretest_questions(story_id, exploration_id):
+    """Checks if an exploration is linked to a story and returns prerequisite
+    skills if so.
+
+    Args:
+        story_id: str. The ID of story linked to the exploration.
+        exploration_id: str. ID of the exploration.
+
+    Returns:
+        list(Question). The list of questions which are the pretests for
+            the exploration, if it is linked to a story, else empty list.
+    """
+    story = get_story_by_id(story_id, strict=False)
+    if story is None:
+        return []
+
+    prerequisite_skill_ids = None
```

Comment:
prerequisite_skill_ids = story.get_prerequisite_skill_ids_for_exp_id(...) (and add unit tests for the new method).

Reply:
Done.